release_notes/v2.2.9.txt

```diff
@@ -0,0 +1 @@
+This is a maintenance release containing bug fixes.
```

Comment:
Right now I'm generally putting text in here that matches what will go in the release description when it is created in GitHub. Look here for examples:
- https://github.com/hyperledger/fabric-sdk-node/blob/release-2.2/release_notes/v2.2.8.txt
- https://github.com/hyperledger/fabric-sdk-node/releases

I am mentioning significant changes or fixes that will be of interest to users, which for this release should probably be the fix to reconnect of event listening:
- https://github.com/hyperledger/fabric-sdk-node/pull/481